### PR TITLE
#12 - creation of meetup info page

### LINKED ISSUE #12: Creation of meetup info page
On meetups list page, after choosing one of them, we should be redirect to page with details about choosen meetup.

There should be:

- title
- information about meetup (short description, date, hour, place, can be photos, etc.)
- link to the event on facebook
- link to page to sign up for the event
- list of presentations, which should contain the following information:
  - title, author, description, tags
  - link to youtube if the event has already taken place and the presentation is published
  - link to the presentation itself
  - the links should rather be in the form of an icon

### PR DESCRIPTION
**Pull request by paulinaszkudlarek**
this should close #12 
Presentations from slideshare are embeded on large screens only
image
image
image

Mobile:
image
image
image
after click on image:
image

### CODE REVIEW

File: backend/ValueObjects/Meetup.php [hunk 1]
```diff
@@ -19,6 +19,7 @@ public function __construct(
         public readonly string $name,
         public readonly string $location,
         public readonly Carbon $date,
+        public readonly string $facebookevent,
```
Comment (EwelinaSkrzypacz) [inline backend/ValueObjects/Meetup.php]:
```suggestion
        public readonly string $facebookEvent,
```

Comment (EwelinaSkrzypacz) [reply backend/ValueObjects/Meetup.php]: Remember to change all occurences.

File: frontend/components/Meetup/MeetupHeader.vue [hunk 2]
```diff
@@ -9,7 +12,7 @@ defineProps({
 
 <template>
   <div class="relative overflow-hidden bg-white">
-    <div class="hidden sm:absolute sm:inset-y-0 sm:block sm:h-full sm:w-full" aria-hidden="true">
+    <div class="z-0 hidden sm:absolute sm:inset-y-0 sm:block sm:h-full sm:w-full" aria-hidden="true">
```
Comment (EwelinaSkrzypacz) [inline frontend/components/Meetup/MeetupHeader.vue]: Do we need `z-0` here?

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1. I see a lot of warnings in dev console:

image

2. I think I will hide gallery, when there is no photo in it:
image

File: frontend/components/Meetup/MeetupHeader.vue (hunk 2 above)
Comment (paulinaszkudlarek) [reply frontend/components/Meetup/MeetupHeader.vue]: nope, removed:)

**Review by EwelinaSkrzypacz — DISMISSED** [review]
For me it's okay - we are waiting for https://github.com/blumilksoftware/laravel-poland-meetup/pull/23 to be merged.

File: backend/ValueObjects/Meetup.php [hunk 3]
```diff
@@ -50,4 +50,4 @@ public function toDetailedEntry(): array
             "facebook" => $this->facebook,
         ];
     }
-}
+}
```
Comment (EwelinaSkrzypacz) [inline backend/ValueObjects/Meetup.php]:
```suggestion
}

```

File: backend/ValueObjects/Person.php [hunk 4]
```diff
@@ -36,4 +36,4 @@ public function toMeetupListEntry()
             "avatar" => "https://ui-avatars.com/api/?background=random&font-size=.33&size-128&rounded=true&name=" . str_replace(" ", "-", $this->name),
         ];
     }
-}
+}
```
Comment (EwelinaSkrzypacz) [inline backend/ValueObjects/Person.php]:
```suggestion
}

```

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1.  The distance between the presenter and the presentation is not equal

image

2. Please check linter.

File: backend/ValueObjects/Person.php (hunk 4 above)
Comment (paulinaszkudlarek) [reply backend/ValueObjects/Person.php]: @EwelinaLasowy Done, all are fixed :)

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by kamilpiech97 — APPROVED** [review]

**Pull request merged**